src/modules/builder/configuration/AddImageStep.tsx

```diff
@@ -45,8 +43,8 @@ const AddImageStep = ({ moveToNextStep }: Props): JSX.Element => {
   const saveData = (): void => {
     const newData = { ...(settings && settings.data), description };
 
-    saveSettings(SettingsKeys.SettingsData, newData);
-    moveToNextStep();
+    saveSettings(SettingsKeys.Settings, newData);
+    setActiveStep(1);
```

Comment:
Should we use an enum here to better describe what'll be the next active step?

```suggestion
    setActiveStep(STEPS.CONFIGURE_STEP); // just an example, didn't remember the next step
```

Reply:
I will just have a "next" function, since currently the tabs are not really related. Might be a good refactor for next time.